src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentForm.java

```diff
@@ -396,5 +460,24 @@ public boolean isDocIsInitiatedOrSaved() {
     public void setDocIsInitiatedOrSaved(boolean docIsInitiatedOrSaved) {
         this.docIsInitiatedOrSaved = docIsInitiatedOrSaved;
     }
+    
+    private boolean isDocInNode(Document document, String node) {
+        WorkflowDocument workflowDocument = document.getDocumentHeader().getWorkflowDocument();
+        Set<String> nodeNames = workflowDocument.getCurrentNodeNames();
+        
+        if (CollectionUtils.isNotEmpty(nodeNames)) {
+            return nodeNames.contains(node);
+        }
+        return false;
+    }
+    
+    private boolean isInOrgHierarchyNode(Document document) {
+        return isDocInNode(document, KFSConstants.RouteLevelNames.ORGANIZATION_HIERARCHY);
+    }
+    
+    private boolean isInPurchasingAssistantNode(Document document) {
```

Comment:
Is this method being called anymore by your updated code? Can it be removed?

Reply:
I am still reworking this. I only committed the changes to be able to deploy to sandbox for testing my changes. I will let you know when this is ready for review. Sorry for the confusion. Thank you.

Reply:
removed